.gitignore

```diff
@@ -8,3 +8,5 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+*.jpg
```

Comment:
There is an extra space before the `*.jpg` entry. Please remove this space to ensure that `.jpg` files are properly ignored by Git.